src/client.ts

```diff
@@ -9320,3 +9337,35 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
  * @event module:client~MatrixClient#"WellKnown.client"
  * @param {object} data The JSON object returned by the server
  */
+
+/**
+ * Metadata on ignored invites.
+ *
+ * This interface is expected to gain additional fields, all
+ * of them optional.
+ */
+export interface IgnoreInviteMetadata {
```

Comment:
All of these interfaces will need a jsdoc line that says they can be removed at any point without notice due to being unstable.